openapi/openapi.yaml

```diff
@@ -1126,6 +1145,9 @@ definitions:
       - published
       - suspended
       - expired
+      - to_test
```

Comment:
`pending_test` ?

Reply:
I preferred test_pending test_passed test_failed

Reply:
Done in https://github.com/pagopa/cgn-onboarding-portal-backend/pull/134/commits/6f467bbf40782949d4abd00f6d83684ee272e870